src/algebra/ordered_monoid.lean

```diff
@@ -562,7 +562,7 @@ le_antisymm bot_le (one_le ⊥)
 @[simp, to_additive] lemma mul_eq_one_iff : a * b = 1 ↔ a = 1 ∧ b = 1 :=
 mul_eq_one_iff' (one_le _) (one_le _)
 
-@[simp, to_additive] lemma le_one_iff_eq : a ≤ 1 ↔ a = 1 :=
+@[simp, to_additive le_zero_iff_eq] lemma le_one_iff_eq : a ≤ 1 ↔ a = 1 :=
```

Comment:
Do you think `nonpos_iff_zero` is a better or worse name here? If `le_zero` is always written `nonpos`, that name feels like it would be more discoverable.

Reply:
I agree that it's a better name and I will make another PR proposing this change. If I change it now then it touches ten files and all of a sudden it's far less clear what this PR is doing.

Reply:
I already assumed this was a new lemma! Thanks for clarifying, reviewing is hard sometimes.

Reply:
oh lol sorry! What's happening here is that we used to have additive lemmas, I changed them to multiplicative ones and then `to_additive` is suggesting additive lemma names which are better than the ones we have right now :-) This happens twice but I thought it might be best not to add the changes to this PR which is already kind of doing a lot.